src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -126,6 +126,14 @@ def _crawl(self) -> Iterable[Result]:
             Iterable[Result]: Records that capture the results of crawling the environment.
         """
 
+    def load_one(self, schema_name: str, table_name: str) -> Result | None:
+        """
+        Placeholder method to using super class for type hints of TableCrawler
+        :param schema_name:
+        :param table_name:
+        :return:
+        """
+
```

Comment:
This isn't really a satisfactory way of solving the problem: it basically says "all crawlers support this method" but for most it doesn't do anything.

Reply:
okay, let's do take two: let's extend the Py4J crawler from `TablesCrawler` and then we can avoid this `fetch_one` hack here. that way, we won't have to change the dependencies in other components that depend on tables crawler.

```
class FasterTableScanCrawler(TablesCrawler):
```